### PR TITLE
feat: Updated design document to include JSON RPC support.

### PR DESCRIPTION
**Pull request by ebadiere**
Design document update.  Add JSON RPC calls to the relay from the `webSocketServer.ts`

**Related issue(s)**: #2117 

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: docs/design/eth-subscribe.md [hunk 1]
```diff
@@ -286,12 +308,13 @@ Add acceptance tests that follow the structure of the existing polling for logs
...
        "gasLimit": "0x47e7c4",
        "gasUsed": "0x38658",
+       "hash":"0x50d2fe6747f21334213a8bc2691f02b6338f265e9f39f12c1f98f247e18f33aa",
        "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
        "miner": "0xf8b483dba2c3b7176a3da549ad41a48bb3121069",
        "nonce": "0x084149998194cc5f",
        "number": "0x1348c9",
        "parentHash": "0x7736fab79e05dc611604d22470dadad26f56fe494421b5b333de816ce1f25701",
-       "receiptRoot": "0x2fab35823ad00c7bb388595cb46652fe7886e00660a01e867824d3dceb1c8d36",
+       "receiptsRoot": "0x2fab35823ad00c7bb388595cb46652fe7886e00660a01e867824d3dceb1c8d36",
```
Comment (acuarica) [inline docs/design/eth-subscribe.md:317]: Both Geth and Infura docs use singular `receiptRoot`

- https://geth.ethereum.org/docs/interacting-with-geth/rpc/pubsub
- https://docs.infura.io/api/networks/ethereum/json-rpc-methods/subscription-methods/eth_subscribe

However, Alchemy uses plural https://docs.alchemy.com/reference/newheads

**Review by acuarica — DISMISSED** [review]
lgtm

File: docs/design/eth-subscribe.md (hunk 1 above)
Comment (ebadiere) [reply docs/design/eth-subscribe.md:319]: Our blocks contain `receiptsRoot`, plural.

**Comment by ebadiere** [comment]
Replaced by #2230

**Pull request closed without merging**